src/main/java/uk/gov/hmcts/befta/exception/InvalidTestDataException.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.befta.exception;
+
+public class InvalidTestDataException extends RuntimeException {
```

Comment:
Could this please extend FunctionalTestException?

Reply:
Sure. Done.